src/lib/processor/process_evict_seat.rs

```diff
@@ -0,0 +1,50 @@
+use std::mem::size_of;
+
+use ellipsis_client::EllipsisClient;
+use phoenix::program::MarketHeader;
+use phoenix_sdk::utils::get_evictable_trader_ix;
+use phoenix_seat_manager::instruction_builders::{
+    create_evict_seat_instruction, EvictTraderAccountBackup,
+};
+use solana_sdk::pubkey::Pubkey;
+
+pub async fn process_evict_seat(
+    client: &EllipsisClient,
+    market_pubkey: &Pubkey,
+    trader_to_evict: &Option<Pubkey>,
```

Comment:
If we're loading the market anyway, why can't this just be inferred?

Reply:
So I was thinking that this would give additional flexibility in case there was a specific trader we wanted to evict, since the SDK just grabs the first one it finds